packages/trader/src/App/app.tsx

```diff
@@ -24,7 +25,19 @@ const TradeModals = Loadable({
 
 const App = ({ passthrough }: Apptypes) => {
     const root_store = initStore(passthrough.root_store, passthrough.WS);
+
     React.useEffect(() => {
+        if (process.env.RUDDERSTACK_STAGING_KEY || process.env.RUDDERSTACK_PRODUCTION_KEY) {
+            Analytics.initialise({
+                growthbookKey: process.env.GROWTHBOOK_CLIENT_KEY,
+                growthbookDecryptionKey: process.env.GROWTHBOOK_DECRYPTION_KEY,
+                rudderstackKey: ['.pages.dev', 'git-fork', 'localhost', 'staging'].some(condition =>
+                    window.location.hostname.includes(condition)
+                )
+                    ? process.env.RUDDERSTACK_STAGING_KEY
+                    : process.env.RUDDERSTACK_PRODUCTION_KEY,
```

Comment:
you dont need to have any condition for production and staging key!
just pass RUDDERSTACK_KEY to it. 
CI will handle the environment variable provision.